multicluster/controllers/multicluster/common/types.go

```diff
@@ -31,7 +31,19 @@ const (
 	InvalidClusterSetID = ClusterSetID("invalid")
 
 	DefaultWorkerCount = 5
+	// LabelIdentityWorkerCount is the numbers of worker used by LabelIdentityReconciler,
+	// LabelIdentityExportReconciler and LabelIdentityResourceImportReconciler.
+	// Using more workers for those reconcilers could have a better performance when a
+	// lot of LabelIdentity events happen concurrently.
+	LabelIdentityWorkerCount = 50
 
 	EndpointIPTypeClusterIP = "ClusterIP"
 	EndpointIPTypePodIP     = "PodIP"
+
+	// ResourceExchangeQPS and ResourceExchangeBurst are used to config the client-go
+	// used by our resource exchange pipeline. Using higher QPS and Burst, instead of
+	// default setting(QPS=5, Burst=10), could significantly improve the performance,
```

Comment:
```suggestion
	// used by Antrea Multi-cluster resource exchange pipeline. Using higher QPS and Burst, instead of
	// default settings, could significantly improve the performance,
```
Remove the default setting values considering it's actually now different values after v1.27.

Reply:
Done.